bindings/nodejs/lib/types/models/api/block-response.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { BlockId, SlotCommitment, SlotIndex } from '../../block';
+
+/**
+ * Block id response.
+ */
+export interface IBlockIdResponse {
+    /**
+     * The block id.
+     */
+    blockId: BlockId;
+}
```

Comment:
No need to have this type in the bindings since we just return the block id, no?